backend/canisters/group_index/impl/src/lib.rs

```diff
@@ -168,6 +171,12 @@ struct Data {
     pub upload_wasm_chunks_whitelist: HashSet<Principal>,
     pub ic_root_key: Vec<u8>,
     pub rng_seed: [u8; 32],
+    #[serde(default = "init_local_group_index_event_sync_queue")]
+    pub local_group_index_event_sync_queue: GroupedTimerJobQueue<LocalGroupIndexEventBatch>,
+}
+
+fn init_local_group_index_event_sync_queue() -> GroupedTimerJobQueue<LocalGroupIndexEventBatch> {
+    GroupedTimerJobQueue::new(3, false)
```

Comment:
Maybe put concurrency higher? 3 is a bit low. If we start using this more and more it would be useful to have it flush them quickly. Maybe 10?